### ISSUE TITLE
linux arm64 support

### ISSUE THREAD

**Issue by annapst** [stackb/bazel-stack-vscode#131]
Hi!
I'm trying to use bazel-stack-vscode VSCode extension on a linux arm64 machine. I'm am seeing the following error:
`~/.vscode-server/data/User/globalStorage/stackbuild.bazel-stack-vscode/file-downloader-downloads/v1.5.3-bzl: 1: Syntax error: "(" unexpected`
When I tried to run the bzl file directly, I got `Exec format error`. Checking with readelf suggests that it is an x84_64 binary instead of arm64.
Is there a way to use the extension on arm64 machine?
Thank you!

**Comment by bluebambu** [comment]
yeah I got the exactly same issue on arm64 machine. Any help is appreciated!

**Comment by AeloriaDev** [comment]
Me too, I meet the same problem

**Comment by chirantan31** [comment]
Me too!
